code/game/machinery/cryopod.dm

```diff
@@ -217,6 +217,11 @@
 	icon_state = base_icon_state
 	find_control_computer()
 
+/obj/machinery/cryopod/examine(mob/user)
+	..(user)
+	if(occupant)
+		to_chat(user, "[occupant] is inside \the [src].")
```

Comment:
I think here `is` should be generic or depend on the occupant. It could be `are`

Reply:
That may lead to grammar issues.

`John Doe are inside the cryopod.`

Reply:
Yeah, I think `is` is the best one to use.

Reply:
`The bees is inside the cryopod.`
:thinking: 
Just use `gender_datums[occupant.gender].is`.